matlab_simulation/05-estimation/fastSLAM/featureSearch_fs.m

```diff
@@ -0,0 +1,36 @@
+%% FUNC to search of features in Field of VIEW
+% INPUTS:(in order of call)
+% -------
+% No of features
+% Map of the Environment
+% Robot's pose at the current time step
+% Maximum Range
+% Maximum FOV
+% OUTPUTS:(in order of call)
+% --------
+% A vector containing all the measured indexs
+% % AUTHOR: BISMAYA SAHOO, EMAIL:bsahoo@uwaterloo.ca
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Feature Search
+    function meas_ind=featureSearch_fs(noFeatures,map,robot_pose,rmax,thmax)
+        meas_ind = [];
+        for i=1:noFeatures
+                if(inview(map(:,i),robot_pose,rmax,thmax))
+                    meas_ind=[meas_ind,i];%concatenate the indices
+                end
+        end
+            function yes = inview(f,x, rmax, thmax)
+            % Checks for features currently in robot's view
```

Comment:
Ok. My bad, I did not realise that there was a global inview function. I understand your concerns. I shall update it to work with the global inview function later, as I am not sure if we are allowed to make changes to our code now.  

I'd be happy to take any more comments that you might have, and incorporate them after the evaluations are done. Thanks!